seatunnel-formats/seatunnel-format-compatible-connect-json/src/main/java/org/apache/seatunnel/format/compatible/kafka/connect/json/CompatibleKafkaConnectDeserializationSchema.java

```diff
@@ -69,12 +71,15 @@ public class CompatibleKafkaConnectDeserializationSchema
     /** Object mapper for parsing the JSON. */
     private final ObjectMapper objectMapper = new ObjectMapper();
 
+    private final CatalogTable catalogTable;
+
     public CompatibleKafkaConnectDeserializationSchema(
             @NonNull SeaTunnelRowType seaTunnelRowType,
             boolean keySchemaEnable,
             boolean valueSchemaEnable,
             boolean failOnMissingField,
-            boolean ignoreParseErrors) {
+            boolean ignoreParseErrors,
+            CatalogTable catalogTable) {
```

Comment:
Why we need both use `seaTunnelRowType` and `catalogTable` at the same time? I think we can replace `seaTunnelRowType` by `catalogTable`.

Reply:
good , comment